Makefile

```diff
@@ -109,8 +110,15 @@ $(FINTO_TOPICS):
 	curl -L "https://drive.usercontent.google.com/download?id=$${FILE_ID}&confirm=true" --progress-bar \
 		-o $@
 
+PROMISES_2023 = data/raw/promises_2023.json
+$(PROMISES_2023): 
+	mkdir -p data/raw
+	FILE_ID=1l0ME3kckgvH-xWPKj6sRIoHhk37NZjFh
+	curl -L "https://drive.usercontent.google.com/download?id=$${FILE_ID}&confirm=true" --progress-bar \
+		-o $@
```

Comment:
meil alkaa koht olee nii mont kertaa sama copypastettu pätkä ni pitäis harkita et saadaaks se jotenki nätimmin abstrahoituu :D